rero_ils/alembic/54134957af7d_loan_checkout_location_pid.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2022 RERO
+# Copyright (C) 2022 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Loan :: add checkout location pid to current ON_LOAN laons."""
+from logging import getLogger
+
+from elasticsearch_dsl import Q
+from invenio_circulation.proxies import current_circulation
+
+from rero_ils.modules.loans.api import Loan, LoansIndexer
+from rero_ils.modules.loans.models import LoanState
+
+# revision identifiers, used by Alembic.
+revision = '54134957af7d'
+down_revision = '90d857fb5c23'
+branch_labels = ()
+depends_on = None
+
+LOGGER = getLogger('alembic')
+indexing_chunck_size = 1000
+
+
+def upgrade():
```

Comment:
Do we need to do something for operation logs?

Reply:
I don't think it's necessary. If OperationLogs are impacted, it seems OK good. If this isn't relevant problem. It will not change the life cycle of a circulation transaction.